core/arch/x86/decode.c

```diff
@@ -2149,7 +2151,10 @@ decode_operand(decode_info_t *di, byte optype, opnd_size_t opsize, opnd_t *opnd)
     }
     case TYPE_K_VEX: {
         /* part of AVX-512: vex.vvvv selects opmask register */
-        *opnd = opnd_create_reg(decode_reg(DECODE_REG_VEX, di, optype, opsize));
+        reg_id_t reg = decode_reg(DECODE_REG_VEX, di, optype, opsize);
+        if (reg == REG_NULL)
+            return false;
+        *opnd = opnd_create_reg(reg);
```

Comment:
Please either describe how this was tested in the commit description, or even better add a test (could pretty quickly add a decode() of a stack buffer to ir_x86.c: e.g., see test_strict_invalid() in ir_x86.c).

Reply:
Done. I've added it to test_strict_invalid, even though it doesn't really test the "strict" flag. Do you want a different function?

Reply:
Works for me.